File: src/main/java/net/hectus/neobb/turn/default_game/structure/glass_wall/TBlueGlassWall.java
package net.hectus.neobb.turn.default_game.structure.glass_wall;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.structure.PlacedStructure;
import net.hectus.neobb.turn.default_game.attributes.clazz.WaterClazz;

public class TBlueGlassWall extends GlassWallTurn implements WaterClazz {
    public TBlueGlassWall(NeoPlayer player) { super(player); }
    public TBlueGlassWall(PlacedStructure data, NeoPlayer player) { super(data, player); }

    @Override
    public String color() {
        return "blue";
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/default_game/structure/glass_wall/TBlueGlassWall.kt
package net.hectus.neobb.modes.turn.default_game.structure.glass_wall

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.matrix.structure.PlacedStructure
import net.hectus.neobb.matrix.structure.StaticStructure
import net.hectus.neobb.matrix.structure.StaticStructures
import net.hectus.neobb.modes.turn.default_game.attribute.clazz.WaterClazz
import net.hectus.neobb.player.NeoPlayer

class TBlueGlassWall(data: PlacedStructure?, cord: Cord?, player: NeoPlayer?) : GlassWallTurn(data, cord, player), WaterClazz {
    override val staticStructure: StaticStructure = StaticStructures.Default.GlassWall.BLUE
}
